functions/pipeline/local.settings.json

```diff
@@ -3,6 +3,14 @@
   "Values": {
     "FUNCTIONS_WORKER_RUNTIME": "python",
     "AzureWebJobsStorage": "{AzureWebJobsStorage}",
-    "STORAGE_CONNECTION_STRING": "replace this when running local func"
+    "STORAGE_CONNECTION_STRING": "",
+    "STORAGE_ACCOUNT_NAME": "",
```

Comment:
@mtarng Just for my understanding: When running functions locally we currently set these env variables. Is adding them explicitly here an alternative to setting environment variables?

Reply:
@andrebriggs exactly. Anything that's set as an environment variable will override what's in this file, but i think we should slowly migrate to only using this file instead of having to add a bunch of stuff as env vars.